src/ocm/components/clusterConfiguration/staticIp/components/FormViewNetworkWide/FormViewNetworkWideFields.tsx

```diff
@@ -180,7 +180,7 @@ export const ProtocolTypeSelect = () => {
   return (
     <>
       <OcmSelectField
-        label="Internet protocol version"
+        label="Networking stack type"
         options={protocolVersionOptions}
```

Comment:
I think we should be applyng this same name change to the components themselves.

eg. `protocolVersionOptions`, `setProtocolType`, etc

Reply:
Feel fee to sync with @ammont82 as I noticed there is another PR modifyng this component, so the renaming can be done there if you prefer
https://github.com/openshift-assisted/assisted-ui-lib/pull/1889/files